INSTALL.md

```diff
@@ -136,116 +177,109 @@ cmake -DCMAKE_PREFIX_PATH=<previous_build_dir>/dependencies/install
-### Complete solution including GUI
+
 Antares source directory is named `[antares_src]` in all commands.
 
-Build can be done 'out of source'.
+Note :
+> cpack NSIS installer creation need a 'out of source build'. The build directory must be outside `[antares_src]` directory
+
+### Complete solution including GUI
 
-You can define build type with ```-DCMAKE_BUILD_TYPE``` option.
+You can define build type with ```-DCMAKE_BUILD_TYPE``` and ```--config``` option.
 
-release
+- release
 
 ```
-cmake -DCMAKE_BUILD_TYPE=release ..
+cmake -B _build -S [antares_src] -DCMAKE_BUILD_TYPE=release
+cmake --build _build --config release
 ```
- debug
+-  debug
  ```
-cmake -DCMAKE_BUILD_TYPE=debug ..
+cmake -B _build -S [antares_src] -DCMAKE_BUILD_TYPE=debug
+cmake --build _build --config debug
 ```
-Note that these are not the standard CMAKE_BUILD_TYPE. CMake files must be updated.
+Note :
+> These are not the standard ``CMAKE_BUILD_TYPE``. CMake files must be updated.
 
 ### Antares Solver and other command line tools (w/o GUI)
 
 Antares Simulator UI application compilation can be disabled at configure time using the option `-DBUILD_UI=OFF` (`ON` by default)
 
 ### Linux using system libs (recommanded)
 - Install dependencies [using package manager](#using-a-package-manager).
-- Create build dir (optionnal but recommanded)
-```
-cd [antares_src]
-mkdir _build
-```
-- Configure build with cmake
 
+- Configure build with cmake
 ```
-cd [antares_src]
-cd _build
-cmake ..
+cmake -B _build -S [antares_src] -DCMAKE_BUILD_TYPE=release
 ```
 - Build
  ```
-cd [antares_src]
-cd _build
-make -j8
+cmake --build _build --config release -j8
 ```
-Note : compilation can be done on several processor with ```-j``` option.
+Note :
+>Compilation can be done on several processor with ```-j``` option.
 
 ### Window using vcpkg (recommanded)
 - Install dependencies [using VCPKG](#using-vcpkg).
 - Choose [vcpkg-triplet]
-- Create build dir (optionnal but recommanded)
-```
-cd [antares_src]
-mkdir _build
-```
+
 - Configure build with cmake
 
 ```
-cd [antares_src]
-cd _build
-cmake -DCMAKE_TOOLCHAIN_FILE=[vcpkg_root]/scripts/buildsystems/vcpkg.cmake -DVCPKG_TARGET_TRIPLET=[vcpkg-triplet] ..
+cmake -B _build -S [antares_src] -DCMAKE_TOOLCHAIN_FILE=[vcpkg_root]/scripts/buildsystems/vcpkg.cmake -DVCPKG_TARGET_TRIPLET=[vcpkg-triplet] -DCMAKE_BUILD_TYPE=release
 ```
 - Build
  ```
-cd [antares_src]
-cd _build
-cmake -build .
+cmake --build _build --config release -j8
 ```
-
+Note :
+> Compilation can be done on several processor with ```-j``` option.
 ### Linux/Window building external librairies
-- Create build dir (optionnal but recommanded)
-```
-cd [antares_src]
-mkdir _build
-```
+
 - Configure build with CMake with ```BUILD_DEPS``` option.
 
 ```
-cd [antares_src]
-cd _build
-cmake -DBUILD_DEPS=ON ..
+cmake -B _build -S [antares_src] -DBUILD_DEPS=ON -DCMAKE_BUILD_TYPE=release ..
 ```
 - Build
  ```
-cd [antares_src]
+cmake --build _build --config release -j8
```

Comment:
Not sure --config release is necessary if -DCMAKE_BUILD_TYPE=release was given at configuration time